rayon-core/src/lib.rs

```diff
@@ -1,5 +1,47 @@
+//! [`Join`]: fn.join.html
+//! [`Scope`]: fn.scope.html
+//! [`Spawn`]: fn.spawn.html
+//! [`Threadpool`]: struct.threadpool.html
+//! [`install()`]: struct.ThreadPool.html#method.install
+//! [`spawn()`]: struct.ThreadPool.html#method.spawn
+//! [`join()`]: struct.ThreadPool.html#method.join
+//! [`ThreadPoolBuilder`]: struct.ThreadPoolBuilder.html
+//!     
+//! Rayon-core houses the core stable APIs of Rayon.
+//!
+//! These APIs have been mirrored in the Rayon crate and it is recommended to use these from there.
+//!
+//! [`Join`] is used to take two closures and potentially run them in parallel.
```

Comment:
Also, capitalization matters for code references -- this should be `join`. You can say "The `join` function" if you're trying to avoid starting a sentence with lowercase.

Same for `scope` and `spawn` references below, and `ThreadPool` should have a capital `P`.

Reply:
Okay thanks!
Considering that join_context, scope_fifo, spawn_fifo are also mirrored in Rayon, should I document those aswell?